proto/cosmos/slashing/v1beta1/slashing.proto

```diff
@@ -43,6 +43,7 @@ message Params {
 
   int64 signed_blocks_window  = 1;
   bytes min_signed_per_window = 2 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
     (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
     (gogoproto.nullable)   = false,
     (amino.encoding)       = "cosmos_dec_bytes",
```

Comment:
@kocubinski I see you've added this annotation in https://github.com/cosmos/cosmos-sdk/pull/15019.
However it uses the same encoder that for the `cosmos.Dec` scalar annotation: https://github.com/cosmos/cosmos-sdk/blob/673270d7bd0072deb17848824d5abb845333c5da/x/tx/signing/aminojson/json_marshal.go#L66

Have you purposely created this annotation instead of using `cosmos.Dec`? And if so why?

Reply:
Updated in https://github.com/cosmos/cosmos-sdk/pull/17000